lib/BackbeatConsumer.js

```diff
@@ -161,6 +147,34 @@ class BackbeatConsumer extends EventEmitter {
         return this;
     }
 
+    _incrementMetrics(entry) {
+        const qEntry = QueueEntry.createFromKafkaEntry(entry);
+        if (!qEntry.error && qEntry.constructor.name === 'ObjectQueueEntry') {
```

Comment:
Commented in the commit itself, reporting here for clarity: you can do `qEntry instanceof ObjectQueueEntry`, which also supports inheritance from ObjectQueueEntry.